common/changes/@itwin/property-grid-react/make_elements_list_sorted_2023-08-23-13-59.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/property-grid-react",
+      "comment": "Sort selected elements list by label in an ascending order. ",
```

I missed one more space. 😞 

```suggestion
      "comment": "Sort selected elements list by label in an ascending order.",
```